### ISSUE TITLE
DJV cant cache Video playback like mov/mp4

### ISSUE THREAD

**Issue by MaxSteven** [darbyjohnston/DJV#455]
With v2.0.8 I tried quicktime animation, quicktime prores and even h264 mp4
All cant be memory cache for playback like sequence exr
I tried with the same file in djv 1.3.0 and everything is fine
So I believe this is a bug.
<img width="1738" alt="%pn-04-02-22-103348-884" src="https://user-images.githubusercontent.com/6467223/152458388-5bc485d3-449a-4932-9953-ab2926df877a.png">

**Comment by travisbutton** [comment]
Yes, would be great if Darby could acknowledge and/or address this for an upcoming release. Definitely a bug, not a feature.

**Comment by darbyjohnston** [comment]
Hi, this is a bug with the new version of DJV, there is also another issue about it here:
https://github.com/darbyjohnston/DJV/issues/66

I actually have a fix for this in a new project called "tlRender", which is a library for building playback actions partially based on the DJV code:
https://github.com/darbyjohnston/tlRender

My intention was to migrate DJV to use the tlRender library, which would not only fix the caching but a number of other issues as well, like multiple audio and video tracks, multiple clips, and A/B comparison. But now with OpenRV and xSTUDIO released I'm not sure if it is worth the effort?

If you would like to try tlRender there are two other player applications currently using it:
https://www.ftrack.com/en/cinesync/download-cinesync-5
https://github.com/ggarra13/mrv2

**Comment by travisbutton** [comment]
Hey @darbyjohnston. Thanks for the reply here. Valid question with regards to OpenRV, the only problem there is I think the percentage of people successfully compiling it through the convoluted process seems to be pretty small based on the GitHub commentary. I tried on 3 different Windows installs and never end up with an executable, even when it "succeeds" without errors.

We already own a couple of standalone RV licenses before Autodesk integrated it into Shotgun/grid or whatever they call it these days but ultimately want a solid player on more workstations. Thanks for the suggestion on tlRender. Will check it out.

**Comment by darbyjohnston** [comment]
Hi, after a long delay I have released a new version, 3.0.0, which uses tlRender for playback. All of the caching issues should be resolved now, but feel free to reopen this if you see any problems with the new version.
